plugins/inputs/sqlserver/azuresqldbqueries.go

```diff
@@ -233,21 +233,24 @@ SELECT
 		WHEN slo.[edition] = 'Hyperscale' then NULL
 		ELSE CAST(DATABASEPROPERTYEX(DB_NAME(),'MaxSizeInBytes') as bigint)/(1024*1024)
 	END AS [total_storage_mb]
-	,CASE
+	,(SELECT SUM(CAST(FILEPROPERTY(name, 'SpaceUsed') AS INT) / 128) FROM sys.database_files WHERE type_desc = 'ROWS') AS used_space_data_mb
+	 ,CASE
 		WHEN slo.[edition] = 'Hyperscale' then NULL
 		ELSE (
-			cast(DATABASEPROPERTYEX(DB_NAME(),'MaxSizeInBytes') as bigint)/(1024*1024) -
-			(select SUM([size]/128 - CAST(FILEPROPERTY(name, 'SpaceUsed') AS int)/128) FROM sys.database_files)
+			SELECT (CAST(DATABASEPROPERTYEX(DB_NAME(), 'MaxSizeInBytes') AS BIGINT) / (1024 * 1024) - SUM(CAST(FILEPROPERTY(name, 'SpaceUsed') AS  INT) / 128))
+			FROM sys.database_files
+			WHERE type_desc = 'ROWS'
 		)
-	END AS [available_storage_mb]
+	  END AS [available_storage_mb]
+	 ,(SELECT SUM(max_size) * 8 / (1024 * 1024) FROM sys.database_files WHERE type_desc = 'LOG') AS total_log_mb
```

Comment:
```suggestion
	 ,(SELECT SUM(max_size) * 8 / (1024 * 1024) FROM sys.database_files WHERE type_desc = 'LOG') AS max_log_storage_mb
```

`total_log_mb` might be a bit ambiguous as a name. It's unclear if it refers to total log used size, total log allocated size, or maximum possible log size. Suggested name attempts to be more explicit. It is also consistent with the naming convention used with other columns (`availaible_storage_mb`, `used_storage_mb`).

Reply:
Makes sense. So to keep consistent, here my proposal:

**max_storage_mb** (to replace **total_storage_mb**) = maximum capacity for data 
**max_log_mb** (to replace **max_log_storage_mb**)  = maximum capacity for transaction log for the database tier
**used_space_data_mb** = space used within data files
**available_storage_mb** (keep this name to avoid breaking change) = Remaining free space that can be allocated regarding the maximum space provisioned for the database tier

Reply:
`total_storage_mb` is an existing column, right? If we are keeping `available_storage_mb` for backward compatibility reasons, shouldn't we do that for `total_storage_mb` too?

The naming convention for all storage-related columns in this query could be improved for consistency and clarity, but in my view, we should prioritize compatibility.

Reply:
My bad, yes total_storage_mb already exists so we can't change it. 
Last commit contains change for total_log_mb -> max_log_mb